app/scripts/script.js

```diff
@@ -0,0 +1,69 @@
+// DateTime Dsplay
+function startTime() {
+    var today = new Date();
+    document.getElementById('datetime').innerHTML =
+     today;
```

Comment:
you can combine these lines
```suggestion
    document.getElementById('datetime').innerHTML = new Date();
```